.github/workflows/automerge.yml

```
@@ -0,0 +1,51 @@
+# this uses the action to merge special branches on push
+# https://github.com/marketplace/actions/merge-branch
+name: Merge protected branches
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ornl-next, release-next]
+
+jobs:
+  merge-relase-into-main:
+    runs-on: ubuntu-latest
+    if: ${{ github.ref == release-next }}
```

Needs quotes around the branch name, and use `ref_name` instead of `ref`
```suggestion
    if: ${{ github.ref_name == 'release-next' }}
```